src/languages/getNodeMatcher.ts

```diff
@@ -65,7 +65,7 @@ const languageMatchers: Record<
   markdown,
   php,
   python,
-  ruby, 
+  ruby: queryBasedSpecification("ruby"),
```

Comment:
Ideally, I would view this as an "overwrite with precedence" for a transition period. 

We could keep the existing scopes and overwrite the ones which are defined within the scheme files. This might require a definition file or a parse of the `.scm` files and then a match against the values in this scopeType->queryName [mapping here](https://github.com/cursorless-dev/cursorless/pull/629/files#diff-f54826399411e7cc52918c4069c886565dd457b9d472d8e40833a0cb2044bd53).

src/util/scopeToKeyword.ts

Reply:
Yep, sounds good, I agree.

Reply:
![image](https://user-images.githubusercontent.com/1354217/167174417-2239d3dc-d555-42fc-924a-8c3a52cf6456.png)
Also adding lang name